docs/known-issues.md

```diff
@@ -8,9 +8,12 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
+| The Unity editor randomly crashes on MacOS when using ECS preview packages. This is not caused by the SpatialOS GDK.                                                                                                                                                                                 | 2018/07/25 | 1064084      | Simply retry.                                                         | No     |
+| IL2CPP does not currently work on Mac as it crashes upon starting the game. This is due to a bug in Unity that does not initialize a system correctly, if one of the injected `IComponentData` contains a `long`.                                                                                    | 2018/08/31 | 1076596      | Use the Mono compiler.                                                | Yes    |
+| `bool` is not a [blittable type](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types), so will cause an error if used to store component data.                                                                                                               | 2018/09/17 | None         | Use `BlittableBool` instead.                                          | No     |
+| Accessing component data from an entity query will segfault, therefore requests where `ResultType` is `SnapshotResultType` will be dropped before sending.                                                                                                                                           | 2018/09/19 | None         | Don't send entity queries where `ResultType` is `SnapshotResultType`. | No     |
+| [Writer [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-update-component-properties): Updating a component using `Writer.Send` will lead to the SpatialOS GDK updating all properties of the component even if you indicated to only partially update a component. | 2018/09/25 | None         | None                                                                  | No     |
+| [Reader [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): The `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property was changed even if it was not truly changed.      | 2018/09/25 | None         | Verify manually whether a component property was actually changed.    | No     |
+| [Reader [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): `Reader.<component property name>Updated` is may be invoked even if `<component property name>` did not truly change.                                                | 2018/09/25 | None         | Verify manually whether a component property was actually changed.    | No     |
```

Comment:
As discussed above, suggest a slight tweak:
(also note typo "is may")

"The GDK may invoke `Reader.<component property name>Updated`, even if the `<component property name>` did not change. "